osmaxx-py/osmaxx/excerptexport/models/output_file.py

```diff
@@ -1,15 +1,18 @@
-from django.utils.translation import ugettext_lazy as _
 import os
 import uuid
 
 from django.db import models
+from django.utils.translation import ugettext_lazy as _
 
 from .extraction_order import ExtractionOrder
+from osmaxx.excerptexport import settings as excerptexport_settings
```

Comment:
I still don't like to have two settings, this is (from a Django perspective) confusing since settings clashes with the settings. If they are settings, they belong into the default location. If they are not, they should be renamed (ie. options, constants, helpers or whatevertheyreallyare).